recipe_scrapers/barefootinthepines.py

```diff
@@ -16,16 +16,41 @@ def ingredient_groups(self):
         )
 
     def nutrients(self):
-        nutrition_container = self.soup.select_one(".mv-create-nutrition-box")
-        if not nutrition_container:
+        nutrition_section = self.soup.select_one(".mv-create-nutrition-box")
+        if not nutrition_section:
             return None
 
-        nutrition_data = {}
-        for item in nutrition_container.select(".mv-create-nutrition-item"):
-            label = item.select_one(".mv-create-nutrition-label")
-            if label:
-                label_text = label.get_text(strip=True)
-                quantity = item.get_text(strip=True).replace(label_text, "").strip()
-                nutrition_data[label_text] = quantity
+        raw_nutrition_data = {
+            item.select_one(".mv-create-nutrition-label")
+            .get_text(strip=True)
+            .lower(): item.get_text(strip=True)
+            .replace(
+                item.select_one(".mv-create-nutrition-label").get_text(strip=True), ""
+            )
+            .strip()
+            for item in nutrition_section.select(".mv-create-nutrition-item")
+            if item.select_one(".mv-create-nutrition-label")
+        }
 
-        return nutrition_data
+        nutrition_label_mapping = {
+            "calories:": "calories",
+            "carbohydrates:": "carbohydrateContent",
+            "cholesterol:": "cholesterolContent",
+            "total fat:": "fatContent",
+            "fiber:": "fiberContent",
+            "protein:": "proteinContent",
+            "saturated fat:": "saturatedFatContent",
+            "serving size:": "servingSize",
+            "sodium:": "sodiumContent",
+            "sugar:": "sugarContent",
+            "trans fat:": "transFatContent",
+            "unsaturated fat:": "unsaturatedFatContent",
+        }
+
+        standardized_nutrition_data = {
+            schema_label: raw_nutrition_data[custom_label.lower()]
+            for custom_label, schema_label in nutrition_label_mapping.items()
+            if custom_label.lower() in raw_nutrition_data
+        }
```

Comment:
Two nitpicky details about the nutritional retrieval:

* If possible, I'd suggest trying to remove the colon (`:`) suffix from each key in the mapping.  They can make for some confusion when reading the dictionary contents (either visually, or I expect if a screenreader speaks the symbols).
* As a small optimization, it may make sense to drive the for-loop from `raw_nutrition_data` (the items we found on the page, which should be a subset of all possible nutrients) rather than from `nutrition_label_mapping` (which could instead serve as a lookup table).

I haven't experimented with those locally yet, so perhaps there's a good reason to structure it this was instead; if so my apologies!